contracts/events/LoanMaintenanceEvents.sol

```diff
@@ -0,0 +1,10 @@
+pragma solidity 0.5.17;
+
+contract LoanMaintenanceEvents {
+    
+    event DepositCollateral(
```

Comment:
Would not be a good idea to add the sender?

Reply:
Victor just asked for loanId and amount, but yes, maybe it'd turnout helpful at some point in the future to have the sender as well.